Dockerfile-manager

```diff
@@ -7,6 +7,7 @@ RUN dnf -y update && dnf install -y dnf-plugins-core && dnf copr enable -y @vmaa
         && rm -rf /var/cache/dnf/*
 
 ENV APPBASEDIR=/manager
+ENV INSIGHTS_VULNERABILITY_VERSION=latest
```

Comment:
we're adding this variable but it doesn't have any use, I'd do it same way as in vmaas https://github.com/RedHatInsights/vmaas/blob/master/webapp/app.py:
```
VULNERABILITY_VERSION = os.getenv("INSIGHTS_VULNERABILITY_VERSION", "unknown")
SPEC = APISpec(
    title='Vulnerability Engine Manager',
    version=VULNERABILITY_VERSION,
    plugins=(
        'apispec.ext.tornado',
    ),
    basePath="/",
)
```

and the `VersionHandler` returning current version using API

Reply:
thanks, good idea.